app/views/static_pages/home.html.slim

```diff
@@ -12,7 +12,7 @@ html
       = render 'shared/home_nav'
       = render 'static_pages/hero'
     = render 'static_pages/content'
-    = render 'static_pages/connect'
+    = render 'static_pages/connect', networks: @networks
```

Comment:
Same here. Didn't we already merge this feature?

Reply:
No

Reply:
You are right.